### PR TITLE
Sync with up-java for 1.5.6

### PR DESCRIPTION
**Pull request by dddj698**
Use ce_name attribute for mapping UAttributes names to CloudEvent names, up-java [PR-89](https://github.com/eclipse-uprotocol/up-java/pull/89),
Use equality check in isRpcMethod() and isRpcResponse(), up-java [PR-91](https://github.com/eclipse-uprotocol/up-java/pull/91),
Add UUID Time based Helper methods, up-java [PR-90](https://github.com/eclipse-uprotocol/up-java/pull/90).

### CODE REVIEW

**Comment by dddj698** [comment]
Rebase after PR22

File: src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt [hunk 1]
```diff
@@ -197,7 +214,9 @@ abstract class UAttributesValidator {
          * @return Returns a [ValidationResult] that is success or failed with a failure message.
          */
         override fun validateType(attributes: UAttributes): ValidationResult {
-            return if (UMessageType.UMESSAGE_TYPE_PUBLISH == attributes.type) ValidationResult.success() else ValidationResult.failure(String.format("Wrong Attribute Type [%s]", attributes.type))
+            return if (UMessageType.UMESSAGE_TYPE_PUBLISH == attributes.type) ValidationResult.success() else ValidationResult.failure(
+                String.format("Wrong Attribute Type [%s]", attributes.type)
+            )
```
Comment (mikaelperrin) [inline src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt:219]: Is this code formatted properly?

File: src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt [hunk 2]
```diff
@@ -216,7 +235,9 @@ abstract class UAttributesValidator {
          * @return Returns a [ValidationResult] that is success or failed with a failure message.
          */
         override fun validateType(attributes: UAttributes): ValidationResult {
-            return if (UMessageType.UMESSAGE_TYPE_REQUEST == attributes.type) ValidationResult.success() else ValidationResult.failure(String.format("Wrong Attribute Type [%s]", attributes.type))
+            return if (UMessageType.UMESSAGE_TYPE_REQUEST == attributes.type) ValidationResult.success() else ValidationResult.failure(
+                String.format("Wrong Attribute Type [%s]", attributes.type)
+            )
```
Comment (mikaelperrin) [inline src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt:240]: Is this code formatted properly?

**Review by mikaelperrin — COMMENTED** [review]
Only some minor comment

File: src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt (hunk 1 above)
Comment (dddj698) [reply src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt]: fixed

File: src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt (hunk 2 above)
Comment (dddj698) [reply src/main/kotlin/org/eclipse/uprotocol/transport/validate/UAttributesValidator.kt]: fixed

**Review by mikaelperrin — APPROVED** [review]

**Pull request merged**